model-optimizer/extensions/ops/range.py

```diff
@@ -69,4 +69,4 @@ def infer(node: Node):
 
         assert start is not None and limit is not None and delta is not None, \
             'Range operation {} with dynamic inputs is not supported'.format(name)
-        node.out_port(0).data.set_value(np.arange(start, limit, delta, dtype=node.soft_get('dtype', start.dtype)))
+        node.out_port(0).data.set_value(np.arange(start, limit, delta, dtype=node.soft_get('output_type', start.dtype)))
```

Comment:
Please, update the Range extender in the MO IR Reader. Make sure that the changes will work for IRs with Range-1.

Reply:
Done. Don't think it will ever work since Range always const-folds.

Reply:
In fact, this might be a bad change. The range never was in any IR.